src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -63,28 +64,9 @@
     HandyReturnValues handyReturnValues = new HandyReturnValues();
 
     private final CapturingMatcher<T> capturingMatcher = new CapturingMatcher<T>();
```

Comment:
Is it possible to use a `new CustomCapturingMatcher(any())` here?  If yes we don't need a `CapturingMatcher ` at all.